src/components/figures/chart/chartJs/chartjs_minor_grid_plugin.ts

```diff
@@ -0,0 +1,69 @@
+import { Chart } from "chart.js";
```

Comment:
Are minor grid line really not a default chartjs option ? Idk if it's worth doing a plugin for that, IMO minor grid lines are kinda useless